app/apps/route.js

```diff
@@ -13,5 +13,10 @@ export default Ember.Route.extend({
     } else {
       return apps;
     }
+  },
+  redirect: function(model) {
+    if(model.get('length') === 0) {
+      this.replaceWith('apps.new', this.modelFor('stack'));
+    }
```

Comment:
perhaps this and the databases routes should also only be redirecting in `index`.

And I'd like to make these `transitionTo` instead of `replaceWith`.